constants.py

```diff
@@ -47,13 +47,21 @@
 """
 
 
-NUMBER_OF_EACH_FOOD_TOKEN = 9
-"""As per the standard rules of the game, there are this many of each standard 
+SPECIAL_FOOD_TYPES = ["wine"]
+"""A list of the special types of food token used for the trail
+
+Each element is a string.
+"""
+
+
+NUMBER_OF_EACH_STANDARD_FOOD_TOKEN = 9
+NUMBER_OF_EACH_SPECIAL_FOOD_TOKEN = 5
+"""As per the standard rules of the game, there are this many of each  
 food token.
 """
 
 
-TOKENS_FOR_TRAIL = {i : NUMBER_OF_EACH_FOOD_TOKEN for i in FOOD_TYPES}
+STANDARD_TOKENS_FOR_TRAIL = {i : NUMBER_OF_EACH_STANDARD_FOOD_TOKEN for i in STANDARD_FOOD_TYPES}
```

Comment:
`i` is typically used to represent integers. I suggest changing to `food` for clarity.